webpack/scenes/ContentViews/Details/Filters/ContentViewFilterDetails.js

```diff
@@ -32,7 +33,7 @@ const ContentViewFilterDetails = () => {
     if (loaded) setDetails(response);
   }, [JSON.stringify(response), loaded]);
 
-  const { type } = details;
+  const { type, inclusion } = details;
```

Comment:
`details` isn't set until after the first `loading` is finished.  Since `useEffect` runs _after_ render, this means there will be at least one render where `details` is `undefined`.  However, in `CVRpmFilterContent.js` the `inclusion` PropType is set to `required`.

I can see two options: (a) either make `inclusion` non-required and set a default prop, or (b) don't render the `<CVRpmFilterContent />` below unless `loading` is true.  I think I prefer (b)

Reply:
This should be fixed.